src/Shared/Microsoft.AspNetCore.Razor.Utilities.Shared/PooledObjects/PooledArrayBuilder`1.cs

```diff
@@ -1333,4 +1333,196 @@ private void MoveInlineItemsToBuilder()
         // Since _inlineCount tracks the number of inline items used, we zero it out here.
         _inlineCount = 0;
     }
+
+    public readonly ImmutableArray<T> ToImmutableOrdered()
```

Comment:
I was looking for these methods just yesterday, and I couldn't find them, but knew I'd seem them somewhere. The mystery is solved! :D